pyproject.toml

```diff
@@ -33,7 +33,7 @@ python = "~3.11.9"
 boto3 = "1.26.86"
 botocore = "1.29.86"
 debugpy = "1.6.7"
-Django = "3.2.19"
+Django = "5.1.1"
```

Comment:
5.1.5 is the latest by the looks of things: would prefer to go to the latest patch

Reply:
I could only find [5.1.4](https://pypi.org/project/Django/5.1.4/)